lib/puppet/indirector/face.rb

```diff
@@ -1,8 +1,8 @@
 require 'puppet/face'
 
 class Puppet::Indirector::Face < Puppet::Face
-  option "--terminus TERMINUS" do
-    summary "The indirector terminus to use."
+  option "--terminus _" + _("TERMINUS") do
+    summary _("The indirector terminus to use.")
     description <<-EOT
```

Comment:
@LarissaLane asked me to leave large docs-like blocks of test untranslated for the time being, due to translation time and budget constraints.